fe/fe-core/src/main/java/org/apache/doris/plugin/AuditEvent.java

```diff
@@ -272,8 +274,49 @@ public AuditEventBuilder setScanBytesFromRemoteStorage(long scanBytesFromRemoteS
             return this;
         }
 
+        public AuditEventBuilder setCommandType(String commandType) {
+            auditEvent.commandType = commandType;
+            return this;
+        }
+
         public AuditEvent build() {
-            return this.auditEvent;
+            AuditEvent copy = new AuditEvent();
+            copy.type = auditEvent.type;
```

Comment:
do not modify this. you should call reset some where when handle EXEUCTE command

Reply:
Thank you for your correction